packages/module/patternfly-docs/content/examples/filterSidePanel.css

```diff
@@ -1,50 +1,46 @@
 .filter-panel-pf {
-  margin: 0 0 30px;
-  padding: 0 15px;
-}
-
-.filter-panel-pf-category {
-  margin-top: 20px;
-}
-
-.filter-panel-pf-category:first-of-type {
-  margin-top: 0;
+  margin-block-start: 0;
+  margin-block-end: var(--pf-t--global--spacer--xl);
+  margin-inline: 0;
+  padding-inline: var(--pf-t--global--spacer--md);
+  display: flex;
+  flex-direction: column;
+  gap: var(--pf-t--global--spacer--lg);
 }
 
 .filter-panel-pf-category-title {
   border: 0;
-  font-size: inherit;
-  margin: 0 0 8px 0;
-  font-weight: 700;
+  font-size: var(--pf-t--global--font--size--body--sm);
+  margin-block-start: 0;
+  margin-block-end: var(--pf-t--global--spacer--sm);
+  margin-inline: 0;
+  font-weight: var(--pf-t--global--font--weight--body--bold);
 }
 
 .filter-panel-pf-category-items {
-  margin-top: 0;
-  margin-bottom: 0;
+  margin-block: 0;
 }
 
 .filter-panel-pf-category-items .pf-v6-c-button.pf-m-link {
   padding: 0;
 }
 
-.filter-panel-pf-category-item {
-  font-weight: 400;
-  margin-bottom: 0;
-  margin-top: 5px;
+.filter-panel-pf-category-items > * + * {
+  margin-block-start: var(--pf-t--global--spacer--sm);
 }
 
-.filter-panel-pf-category-item:first-of-type {
-  margin-top: 0;
+.filter-panel-pf-category-item {
+  font-weight: var(--pf-t--global--font--weight--body);
 }
 
-.filter-panel-pf-category-item .pf-v6-c-check__input {
-  margin-left: 0px;
+.filter-panel-pf-category-item:first-of-type {
+  margin-block-start: 0;
 }
 
 .filter-panel-pf-category-item .item-icon {
-  padding-right: 3px;
+  padding-inline-end: var(--pf-t--global--spacer--xs);
```

Comment:
It seems like this spacer/padding and the one on `.item-count` below could just come from regular space characters in the markup. The figma designs just use a space between the text and count.